### PR TITLE
fix(UI): Do not allow scroll in Processor Dialog

### PR DESCRIPTION
**Pull request by dsvanlani**
### Proposed Change
This PR changes some styles to allow for the form in the ProcessorDialog to scroll, not the dialog itself.

#### All Processors View
image
#### Select View
image
#### Configure View
image
#### Edit View
image

##### Checklist
- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: ui/src/components/RecentTelemetryDialog/RecentTelemetryDialog.tsx [hunk 1]
```diff
@@ -37,13 +59,19 @@ const RecentTelemetryBody: React.FC<{
...
-        metrics={metrics}
-        traces={traces}
-        footer={footer}
-      />
+    <DialogContainer
+      title="Recent Telemetry"
+      description="Showing a snapshot of recent telemetry taken before it is sent to a destination"
+      onClose={handleClose}
+    >
+      <Stack flexGrow={1} height="100%" className="blah">
```
Comment (andykellr) [inline ui/src/components/RecentTelemetryDialog/RecentTelemetryDialog.tsx:67]: Do we need that className?

File: ui/src/components/ResourceDialog/ProcessorsDialog/InlineProcessorContainer.tsx [hunk 2]
```diff
@@ -84,18 +84,18 @@ export const InlineProcessorContainer: React.FC<Props> = ({
...
-            <Button
-              variant="text"
-              startIcon={<PlusCircleIcon />}
-              classes={{ root: mixins["mb-2"] }}
-              onClick={onAddProcessor}
-            >
-              Add processor
-            </Button>
-          </Stack>
+          // <Stack alignItems="center" className="blah">
```
Comment (andykellr) [inline ui/src/components/ResourceDialog/ProcessorsDialog/InlineProcessorContainer.tsx:87]: Safe to remove?

Comment (dsvanlani) [reply ui/src/components/ResourceDialog/ProcessorsDialog/InlineProcessorContainer.tsx:87]: 👍

File: ui/src/components/RecentTelemetryDialog/RecentTelemetryDialog.tsx (hunk 1 above)
Comment (dsvanlani) [reply ui/src/components/RecentTelemetryDialog/RecentTelemetryDialog.tsx:67]: Negative, good catch thanks.

**Comment by shazlehu** [comment]
Getting some wonkiness:
image

**Comment by shazlehu** [comment]
First view:
image
Then when I scroll down:
image

Should the two sides should scroll independently?

**Review by shazlehu — APPROVED** [review]

**Pull request merged**